geode-core/src/jmh/java/org/apache/geode/internal/logging/log4j/FastLoggerDisabledLevelBenchmark.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.logging.log4j;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.apache.logging.log4j.Level.INFO;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@Measurement(iterations = 1, time = 1, timeUnit = SECONDS)
```

Comment:
If your iterations and time are so low are you certain the JIT is coming into play on these benchmarks? Or is this just a hold over from you testing that this bench still compiles and runs?

Reply:
I set them small just for quick runs, but I do increase the values for various runs. The results of the longer runs are pretty much the same but if I run for 3+ hours then the results tend to be much more consistent than if you run several 1 second runs and compare them all. We can increase the configured values that are checked in if you prefer.

Reply:
Not necessary, just wanted to understand and make sure we evaluated longer runs.